server/src/app.js

```diff
@@ -1,11 +1,25 @@
 import express from 'express';
+import swaggerUi from 'swagger-ui-express';
+import swaggerDocument from '../../swagger.json';
+import sendError from './middlewares/errorHandler';
+import routes from './routes/index';
 
 const app = express();
 
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
+app.use(sendError);
```

Comment:
The error middleware should be the last thing you register on the express app. Is there a reason you registered it here?

Reply:
you are right,it be the last middleware to register